types/simulation/rand_util.go

```diff
@@ -27,17 +28,15 @@ func RandStringOfLength(r *rand.Rand, n int) string {
 		if remain == 0 {
 			cache, remain = r.Int63(), letterIdxMax
 		}
-
 		if idx := int(cache & letterIdxMask); idx < len(letterBytes) {
 			b[i] = letterBytes[idx]
 			i--
 		}
-
 		cache >>= letterIdxBits
 		remain--
 	}
 
-	return string(b)
+	return *(*string)(unsafe.Pointer(&b))
```

Comment:
this had the best performance according to the linked [stackoverflow issue](https://stackoverflow.com/questions/22892120/how-to-generate-a-random-string-of-a-fixed-length-in-golang#31832326)

Reply:
could we add this link into the code? I could see this being reverted real fast without context for why it is this